app/components/publications/new-publication-modal.hbs

```diff
@@ -78,7 +78,6 @@
           {{else}}
             <Auk::Datepicker
               @date={{this.decisionDate}}
-              @disabled={{this.isViaCouncilOfMinisters}}
```

Comment:
Is dit de standaard aanpak: hide inputveld en toon span/div indien niet editeerbaar?

Reply:
Nee, hier is niet echt een standaard aanpak voor. Ik heb het hier aangepast omdat het als `span` ook in de sidepanel getoond wordt [in het prototype](http://master.kaleidos-prototype.mono.digital/shared/publications/detail/dossier--empty.html). Met een disabled state gaan gebruikers misschien ook denken dat ze soms de datum wel zullen kunnen aanpassen (vb. nadat ze andere velden in het formulier ingevuld hebben) terwijl ze in het geval van publicatie via MR dat veld nooit zullen kunnen aanpassen in deze modal.